final_task/rss_reader/functions/check_func.py

```diff
@@ -0,0 +1,85 @@
+"""Contain checking functions
+
+Functions
+---------
+check_internet_connection(logger) -> Bool
+    Check if network connection is avaliable
+--------------------------------------------
+check_version_argument(command_line_args) -> None
+    If version argument, print version and exit
+--------------------------------------------
+check_verbose(command_line_args): -> logger
+    If vebose argument is True, set logger to info level
+--------------------------------------------
+check_feed_status(feed, logger)
+    Check HTTP status code
+--------------------------------------------
+def check_news_collection(news_collection, logger) -> news_collection
+    Check if news_collection is not empty
+"""
```

Comment:
Same as above, it is better to move this to documentation. You do not need this here if you have docstrings in corresponding functions.

Reply:
Fixed